packages/tools/test/apm.coffee

```diff
@@ -29,7 +29,18 @@ describe 'tools.apm', ->
       ssh: ssh
     .system.execute
       cmd: "if (apm list | grep package-list) then (exit 0) else (apm uninstall package-list) fi"
```

Comment:
The all point of writting an unsinstall action is to remove this.

Reply:
Updating with uninstall action.